domains/catzboy.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Catzboy Profilo",
+  "repo": "https://github.com/Maruful007/Maruful007",
+  "owner": {
+    "username": "Maruful007",
+    "email": "marufulislamsami2007@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://maruful007.github.io/Catzboy-Profilo/public/",
```

Comment:
cname shouldnt be an url

Reply:
i want to catzboy.is-a.dev to look same as https://maruful007.github.io/Catzboy-Profilo/public/ what record should i put